README.md

```diff
@@ -22,8 +22,17 @@ apt install qemu-system-x86
 ```
 
 ### Windows
-1. Install QEMU from [here](https://qemu.weilnetz.de/w64/)
-2. Add qemu installation directory to PATH
+* Install QEMU from [here](https://qemu.weilnetz.de/w64/)
```

Comment:
use `-` instead of `*` for consistency with the rest of the readme :)

Reply:
:)